### PR TITLE
[IMP]10697 add module website_forum_nutrition

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: website_forum_nutrition/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            'validation_email_sent': request.session.get(
+                'validation_email_sent', False),
+            'validation_email_done': request.session.get(
+                'validation_email_done', False),
+        }
+        values['header']['ask_hide'] = True
+        if forum:
+            values['forum'] = forum
+        elif kwargs.get('forum_id'):
+            values['forum'] = request.registry['forum.forum'].browse(request.cr, request.uid, kwargs.pop('forum_id'), context=request.context)
```
Comment (elicoidal) [inline website_forum_nutrition/controllers/main.py]: pep8?

File: website_forum_nutrition/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            'forum': forum,
+            'searches': {'symptom': True},
+            'notifications': self._get_notifications(),
+            'pager': pager,
+        })
+
+        return request.website.render(
+            "website_forum_nutrition.symptoms", values)
+
+    @http.route(['/<model("forum.forum"):forum>/<model("product.symptom"):symptom>/products',
```
Comment (elicoidal) [inline website_forum_nutrition/controllers/main.py]: PEP8

**Comment by elicoidal** [comment]
no unit test but I think this is OK.
Other than that small details

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
LGTM, no technical tests. Isnot it duplicated with some I have seen days ago?

**Comment by Rona111** [comment]
@elicoidal  isnot it duplicated with some I have seen days ago? what do you mean?you mean no technical tests is ok or not?

**Comment by elicoidal** [comment]
This PR is similar to another one I have reviewed days ago.
not important though

File: website_forum_nutrition/README.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    -> `Symptom`, create new symptoms as you need.
+ * Go to `product template` form, assign the symptoms as you need.
+
+Known issues / Roadmap
+======================
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline website_forum_nutrition/README.rst]: modify link

File: website_forum_nutrition/README.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20website_forum_nutrition%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline website_forum_nutrition/README.rst]: remove this line

File: website_forum_nutrition/README.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+* Alex Duan <alex.duan@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
```
Comment (elicoidal) [inline website_forum_nutrition/README.rst:58]: use newest template

**Review by elicoidal — CHANGES_REQUESTED** [review]
update the README

File: website_forum_nutrition/README.rst (hunk 5 above)
Comment (noahzaozao) [reply website_forum_nutrition/README.rst:58]: done

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Review by elicoidal — APPROVED** [review]

**Pull request merged**